docs/architecture/blazor-for-web-forms-developers/config.md

```diff
@@ -1,15 +1,187 @@
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <location path="." inheritInChildApplications="false">
+    <system.webServer>
+      <handlers>
+        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
+      </handlers>
+      <aspNetCore processPath=".\MyApp.exe"
+                  stdoutLogEnabled="false"
+                  stdoutLogFile=".\logs\stdout"
+                  hostingModel="inprocess" />
+    </system.webServer>
+  </location>
+</configuration>
+```
+
+Configuration specific to your application can be defined by nesting an `environmentVariables` element inside of the `aspNetCore` element.  The values defined in this section are presented to the ASP.NET Core application as environment variables, and load appropriately during that segment of the application startup.
+
+```xml
+<aspNetCore processPath="dotnet"
+      arguments=".\MyApp.dll"
+      stdoutLogEnabled="false"
+      stdoutLogFile=".\logs\stdout"
+      hostingModel="inprocess">
+  <environmentVariables>
+    <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Development" />
+    <environmentVariable name="Parent:ApiKey" value="67890" />
+  </environmentVariables>
+</aspNetCore>
+```
+
+## Reading Configuration in the Application
+
+ASP.NET Core provides application configuration through an interface at `Microsoft.Extensions.Configuration.IConfiguration`.  This configuration interface should be requested by your Blazor components, Blazor pages, and any other ASP.NET Core managed class that needs access to configuration.  The ASP.NET Core framework will automatically populate this interface with the resolved configuration configured earlier.  On a Blazor page or component's razor markup you can inject the IConfiguration object with an `@inject` statement at the top of the `.razor` file like this:
+
+```cshtml
+@inject IConfiguration Configuration
+```
+
+This statement will make the `IConfiguration` object available as the `Configuration` varialble through the rest of the razor template.
+
+Individual configuration settings can be read by specifying the entire hierarchy of the configuration setting sought as an indexer parameter:
+
+```csharp
+var mySetting = Configuration["section1:key0"]
+```
+
+You can fetch entire sections of the configuration by using the `IConfiguration.GetSection` command to retrieve a collection of keys at a specific location with a syntax similar to: `GetSection("section1")` to retrieve the configuration for section1 from the earlier example.
+
+## Strongly-Typed Configuration
+
+With Web Forms, it was possible to create a strongly-typed configuration type that inherited from the [ConfigurationSection](https://docs.microsoft.com/en-us/dotnet/api/system.configuration.configurationsection?view=netframework-4.8) type and associated types.  This would allow you to configure some business rules and processing for these configuration values.  
```

Comment:
```suggestion
With Web Forms, it was possible to create a strongly-typed configuration type that inherited from the <xref:System.Configuration.ConfigurationSection> type and associated types.  This would allow you to configure some business rules and processing for these configuration values.  
```